include/mgba-util/formatting.h

```diff
@@ -20,8 +20,8 @@ typedef const char* locale_t;
 
 int ftostr_l(char* restrict str, size_t size, float f, locale_t locale);
 
-#ifndef HAVE_STRTOF_L
-float strtof_l(const char* restrict str, char** restrict end, locale_t locale);
```

Comment:
This will break a lot of stuff. I have no idea what you're even using it for here?